src/components/Editor/GeneralInformation/DatabaseWarning.vue

```diff
@@ -113,6 +113,13 @@
           pressYes(){
             this.answered = true;
             this.submitted += 1;
+          },
+          pressNo(){
+            this.showOverlay = false;
+            if (this.submitted === 1) {
```

Comment:
```suggestion
            if (this.submitted >= 1) {
```
=> so if a user presses more than once, it also sends the email :)